linker/shared/src/main/scala/org/scalajs/linker/standard/CoreSpec.scala

```diff
@@ -96,6 +96,9 @@ final class CoreSpec private (
       targetIsWebAssembly
     )
   }
+
+  val linkTimeProperties = new LinkTimeProperties(
+    semantics, esFeatures, targetIsWebAssembly)
```

Comment:
```suggestion
  val linkTimeProperties = new LinkTimeProperties(
      semantics, esFeatures, targetIsWebAssembly)
```
Also, perhaps make it a `lazy val`?